bb-pilcom/bb-pil-backend/src/prover_builder.rs

```diff
@@ -25,6 +25,8 @@ impl ProverBuilder for BBFiles {
         using Flavor = {name}Flavor;
         using FF = Flavor::FF;
         using PCS = Flavor::PCS;
+        using Curve = Flavor::Curve;
+        using ZeroMorph = ZeroMorphProver_<Curve>;
         using PCSCommitmentKey = Flavor::CommitmentKey;
         using ProvingKey = Flavor::ProvingKey;
         using Polynomial = Flavor::Polynomial;
```

Comment:
Why do not we see the diff related to: https://github.com/AztecProtocol/aztec-packages/pull/7078/files#diff-35008898e6a351dcc8432c82b548561d8a581d1e638db6a7b28236e2404c3815L964-R978